### ISSUE TITLE
Use WebRTC for net.createServer

### ISSUE THREAD

**Issue by martindale** [emersion/net-browserify#3]
It seems `net.createServer` is not implemented; WebRTC might serve a reasonable fill-in.

**Comment by emersion** [comment]
I don't really understand how do you want to implement `net.createServer()`?

By creating a WebRTC receiver on one client, and allowing other clients to connect to it?

**Comment by martindale** [comment]
Yes, precisely!  `net.createServer()` is very feasible if implemented via WebRTC.

**Comment by emersion** [comment]
This could be fun, but I don't see how this could be used in a real-world situation... :-P

Another way to implement this would be to listen to a port with the proxy.

**Comment by martindale** [comment]
I'm trying to create an isomorphic peer-to-peer application that doesn't require any servers at all.  `net-browserify` has gotten me quite far down the path, but the final thing remaining is being able to _receive_ connections (even if it requires a third node to coordinate).
